### PR TITLE
[DE] Add Skinning

### PR DESCRIPTION
**Pull request by SorryAward04**
Translated main article for the skinning section to German.
Probably needs review for ASC compliance.
Other articles to be translated soon.

### CODE REVIEW

**Comment by zeusminus** [comment]
Well, it appears that your PR not only adds a German locale for the `Skinning` article, it also removes the entire English locale, you see. Probably that's because you simply worked on the `en.md` and forgot to create a new file for the new locale.

**Comment by SorryAward04** [comment]
> Well, it appears that your PR not only adds a German locale for the `Skinning` article, it also removes the entire English locale, you see. Probably that's because you simply worked on the `en.md` and forgot to create a new file for the new locale.

oh, you are right about that. how should i respond to this mistake?

**Comment by zeusminus** [comment]
> oh, you are right about that. how should i respond to this mistake?

You should restore the `en.md` file on your fork. I guess the easiest way would be to create a new `en.md` folder there and copy-paste the contents from `ppy:master`.

**Comment by TPGPL** [comment]
Restored the file for you, now we only have to wait for the native review ¯\_(ツ)_/¯

File: wiki/Skinning/de.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Skinning
+
+Das Skinning ist eines der Schlüsselmerkmale in osu!. Es ermöglicht Spielern, von den ursprünglichen Skinelementen abzuweichen, um ihre eigenen zu erstellen! Skins können von spaßeshalber, thematisch zu PRO variieren oder nahezu unmöglich zu spielen sein.
```
Comment (Phippe) [inline wiki/Skinning/de.md:3]:
```suggestion
Das Skinning ist eines der Hauptmerkmale in osu!. Es ermöglicht Spielern, aus den ursprünglichen Skinelementen ihre eigenen zu erstellen! Skins können variieren, seien sie nun spaßeshalber, thematisch, pro oder nahezu unmöglich zu spielen.
```

The last sentence is a weird one. I tried to come up with something, but I'm not sure if it's better than yours.

File: wiki/Skinning/de.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Skinsets
+
+*Haupseite: [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**Nur bei Beatmaps,** falls deine Beatmap ein einzelnes Element aus den aufgezählten Sets in den Rangfolgekriterien enthält, muss es alle anderen Elemente jenes besagten Skinsets enthalten. Dies bezieht sich nur auf Beatmapskins, jedoch mögen andere Skinner diese Listen ebenfalls in Erwägung ziehen wollen.
```
Comment (Phippe) [inline wiki/Skinning/de.md:9]:
```suggestion
**Nur bei Beatmaps:** Falls der Skin deiner Beatmap ein einzelnes Element aus den in den Ranking-Kriterien aufgezählten Sets enthält, muss er alle anderen Elemente jenes besagten Skinsets enthalten. Dies bezieht sich nur auf Beatmapskins, jedoch mögen andere Skinner diese Listen ebenfalls in Erwägung ziehen wollen.
```

**Comment by TicClick** [comment]
@SorryAward04 do you plan to return to this PR and accept the suggestions?
(by the way, @Phippe, do you have anything else to comment on?)

**Comment by Phippe** [comment]
@TicClick No, that's it I think. The first suggestion really is just a suggestion; to me the translation feels a bit halting/rough. However, the second suggestion is necessary due to the current translation having a different meaning from the original text (_beatmap skin_ was just translated as _beatmap_).

**Comment by SorryAward04** [comment]
I am back after forever, these recommendations will be applied.
Also thank you for the file restoration @TPGPL, I appreciate it.

**Comment by TicClick** [comment]
> I am back after forever, these recommendations will be applied.

so?

File: wiki/Skinning/de.md (hunk 1 above)
Comment (Okorin) [inline wiki/Skinning/de.md:3]:
```suggestion
Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen, um ihre eigenen zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, fokussiert auf "pro-Spielbarkeit", Minimalismus, oder nahezu unspielbar.
```

File: wiki/Skinning/de.md (hunk 2 above)
Comment (Okorin) [inline wiki/Skinning/de.md:9]:
```suggestion
Wenn du einen speziellen Skin in deiner Beatmap verwendest, wird dies als Beatmap-spezifisches Skinning bezeichnet. Hierbei werden die einzelnen Elemente des Skins direkt in den Beatmap-Ordner gepackt und dadurch im Spiel verwendet sofern der Spieler dies nicht ausdrücklich anders eingestellt hat. Wenn du einen solchen Skin verwenden möchtest, ist das [Ranking Criteria](/wiki/Ranking_Criteria) sowie die [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List) zu beachten. 

Falls dein Beatmap-Skin ein gameplay-relevantes Element enthält, müssen alle obligatorischen Elemente aus der zugehörigen Skinsetliste ebenfalls in deiner Beatmap enthalten sein. Genau genommen beziehen sich diese Einschränkungen nur auf Beatmap-Skins für Beatmaps, die den Ranking-Prozess durchlaufen, aber es ist als Skinner generell sinnvoll, diese Liste beim Erstellen eines Skins in Erwägung zu ziehen.
```

**Review by Okorin — CHANGES_REQUESTED** [review]
I changed your initial translation to sound more coherent and provided a few more comments that are based on the currently available Ranking Criteria for that matter. Part of this original article was wrong and I thought it was generally too difficult to just understand what beatmap-specific skinning was on its own so I added a paragraph elaborating on what it actually is.

File: wiki/Skinning/de.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Skinning
+
+Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen, um ihre eigenen zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, fokussiert auf "pro-Spielbarkeit", Minimalismus, oder nahezu unspielbar.
+
+## Skinsets
+
+*Haupseite: [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+Wenn du einen speziellen Skin in deiner Beatmap verwendest, wird dies als Beatmap-spezifisches Skinning bezeichnet. Hierbei werden die einzelnen Elemente des Skins direkt in den Beatmap-Ordner gepackt und dadurch im Spiel verwendet sofern der Spieler dies nicht ausdrücklich anders eingestellt hat. Wenn du einen solchen Skin verwenden möchtest, ist das [Ranking Criteria](/wiki/Ranking_Criteria) sowie die [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List) zu beachten.
```
Comment (Okorin) [inline wiki/Skinning/de.md:9]: uhhh i think i need a comma between
`verwendet, sofern `

but am not sure

Comment (Feerum) [reply wiki/Skinning/de.md:9]: I'd put a comma between "verwendet" and "sofern". So basically: [...] und dadurch im Spiel verwendet, sofern der Spieler dies nicht [...]

Comment (Okorin) [inline wiki/Skinning/de.md:9]:
```suggestion
Wenn du einen speziellen Skin in deiner Beatmap verwendest, wird dies als Beatmap-spezifisches Skinning bezeichnet. Hierbei werden die einzelnen Elemente des Skins direkt in den Beatmap-Ordner gepackt und dadurch im Spiel verwendet, sofern der Spieler dies nicht ausdrücklich anders eingestellt hat. Wenn du einen solchen Skin verwenden möchtest, ist das [Ranking Criteria](/wiki/Ranking_Criteria) sowie die [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List) zu beachten. 
```

**Review by TicClick — DISMISSED** [review]
thanks to all participants

File: wiki/Skinning/de.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Skinning
+
+Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen, um ihre eigenen zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, fokussiert auf "pro-Spielbarkeit", Minimalismus, oder nahezu unspielbar.
```
Comment (Feerum) [inline wiki/Skinning/de.md:3]:
```suggestion
Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen oder ihre eigenen Skins zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, fokussiert auf "pro-Spielbarkeit", Minimalismus, oder nahezu unspielbar.
```

Something reads weird here at the beginning. Like something was missing

**Review by Okorin — APPROVED** [review]
less clunky now

**Pull request merged**